.tekton/build_and_trigger.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     - name: event_type
       value: "{{ event_type }}"
     - name: registry-url
-      value: "quay.io/redhat-appstudio"
+      value: "quay.io/konflux-ci"
     - name: registry-username
       value: "{{ registry_username }}"
     - name: registry-password
```

Comment:
How do registry_username and registry_password get sourced?

Reply:
I _think_ this gets injected by PaC, but not sure. @jencull or @jsztuka you may know better than me about how this is set up.

Reply:
I think you would need to add robot account to the https://quay.io/repository/konflux-ci/clair-in-ci?tab=info.
robot account: konflux-ci+testservice

Reply:
IIRC username and password are not being used at all, robot account is doing all the magic.

Reply:
Service account should be using those secrets:
https://github.com/redhat-appstudio/infra-deployments/blob/main/components/konflux-ci/base/external-secrets/clair-in-ci-db-github-token.yaml
We need to update those.

Reply:
@Josh-Everett robot user should be added to to [writers group ](https://quay.io/organization/konflux-ci?tab=teams)  then writers group be added to the [repo clair-in-ci](https://quay.io/repository/konflux-ci/clair-in-ci?tab=info) .

Reply:
AFAIK Scott added that robot account to the quay repo, but here it is still set to use the old user + pass. I am having trouble figuring out where to set which robot account to use.

Reply:
I guess we will need some authorization to be able seeing what members in our integration repos 
so we can solve such these cases